packages/@lwc/engine-core/src/framework/decorators/api.ts

```diff
@@ -75,6 +76,21 @@ export function createPublicPropertyDescriptor(key: string): PropertyDescriptor
     };
 }
 
+const DUMMY_ACCESSOR_REACTIVE_OBSERVER = {
```

Comment:
I'm not a fan of having this here... this API is more suitable for the actual place where we implement it. Can we move this into that other file?

Reply:
Sure, moved `AccessorReactiveObserver` into its own file. (I didn't want to bloat `membrane.ts`.